### PR TITLE
Send Button remains enabled all time even when the text is not present in the textbox

### PR DESCRIPTION
**Pull request by cmevawala**
## Description
When the send button component is getting created, we need to check whether there is any value in textbox or not and based on that set the disabled property

## Specific Changes
Changes are done only in SendButton.js where we get the **sendBoxValue** and we used that value to set the disabled property

### CODE REVIEW

File: packages/component/src/SendBox/SendButton.js [hunk 1]
```diff
@@ -14,16 +14,21 @@ const connectSendButton = (...selectors) =>
...
     ...selectors
   );
 
-const SendButton = ({ click, disabled, language }) => (
-  <IconButton alt={localize('Send', language)} disabled={disabled} onClick={click}>
-    <SendIcon />
-  </IconButton>
-);
+const SendButton = ({ click, disabled, language, sendBoxValue }) => {
+  disabled = sendBoxValue ? false : true;
```
Comment (corinagum) [inline packages/component/src/SendBox/SendButton.js]: My concern is that this is overwriting the props value `disabled`. @compulim, thoughts?

**Comment by corinagum** [comment]
Looking for design input from @DesignPolice

**Comment by corinagum** [comment]
Hi @cmevawala, unfortunately I don't think this code is achieving what you intend. These changes will actually disable the send box and prevent a user from entering any message to the bot. Please take a look at the following link: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_input_disabled

I am closing this issue since this is not a change we want to implement.

File: packages/component/src/SendBox/SendButton.js (hunk 1 above)
Comment (cmevawala) [reply packages/component/src/SendBox/SendButton.js]: @corinagum : Here are snapshots of the issue which i have fixed

image

image

**Comment by compulim** [comment]
I think the better fix is:

```jsx
<IconButton alt={localize('Send', language)} disabled={disabled || !sendBoxValue} onClick={click}>
```

**Pull request closed without merging**